content/debugging/docs/connect-spark-ui.md

```diff
@@ -0,0 +1,69 @@
+# **Connect to Spark UI running on the Driver Pod**
+
+To obtain more detail about their application or monitor their job execution, Spark application developers can connect to Spark-UI running on the Driver Pod.
```

Comment:
Could you add a statement that, Spark UI (Spark history server) is packaged with EMR on EKS out of the box. Alternatively, if customers want to see Spark UI immediately after the driver is spun up, they can use `kubectl port-forward` route?